content-dpr/tiled-background-image-with-content-dpr.html

```diff
@@ -0,0 +1,27 @@
+<html>
+  <head>
+    <title>Content-DPR: css tiled background</title>
+    <link rel="author" title="Noam Rosenthal" href="noam@webkit.org">
+    <link rel="match" href="tiled-background-image-with-content-dpr-ref.html" />
+    <meta name="assert" content="Assert that content-dpr is taken onto account for tiled background images">
+    <style>
+      #bg, #bg2 {
+        width: 32px;
+        height: 32px;
+        background-image: url(resources/dpr.py?name=background.png&mimeType=image/png&dpr=2.0);
+      }
+
+      #bg2 {
+        background-size: 8px 8px;
```

Comment:
Did you mean to also include the image here?

Reply:
It's there in the previous CSS declaration

Reply:
(same image, just with/without background-size)

Reply:
you're right, of course! Missed it earlier